src/regtech_api_commons/api/exception_handlers.py

```diff
@@ -18,15 +17,15 @@ async def regtech_http_exception_handler(request: Request, exception: RegTechHtt
     log.error(exception, exc_info=True, stack_info=True)
     return JSONResponse(
         status_code=exception.status_code,
-        content={ERROR_NAME: exception.name, ERROR_DETAIL: jsonable_encoder(exception.detail)},
+        content={ERROR_NAME: exception.name, ERROR_DETAIL: str(exception.detail)},
     )
 
 
 async def request_validation_error_handler(request: Request, exception: RequestValidationError) -> JSONResponse:
     log.warn(exception, exc_info=True, stack_info=True)
     return JSONResponse(
         status_code=HTTPStatus.UNPROCESSABLE_ENTITY,
-        content={ERROR_NAME: "Request Validation Failure", ERROR_DETAIL: jsonable_encoder(exception.errors())},
+        content={ERROR_NAME: "Request Validation Failure", ERROR_DETAIL: str(exception.errors())},
```

Comment:
fastapi's documentation is inaccurate / there is a bug in fastpi: https://fastapi.tiangolo.com/tutorial/handling-errors/#override-request-validation-exceptions
```python
str(exception)
```
returns blank instead of the nice text they showed:
```
1 validation error
path -> item_id
  value is not a valid integer (type=type_error.integer)
```
so just stringifying the sequence for now